src/fsharp/pars.fsy

```diff
@@ -286,7 +286,7 @@ let rangeOfLongIdent(lid:LongIdent) =
 %type <Ast.SynExceptionDefn> exconDefn
 %type <Ast.SynExceptionDefnRepr> exconCore
 %type <Ast.SynModuleDecl list> moduleDefnsOrExprPossiblyEmptyOrBlock
-%type <Ast.LongIdentWithDots> openDecl
+%type <Ast.LongIdentWithDots option> openDecl
```

Comment:
Hmm. Doesn’t this change the F# grammar, allowing empty open statements? If so, is it ok?

Reply:
Later on when we check the option, if it is None, we throw an error, "Identifier expected", so it doesn't pass-through.